environment.yml

```diff
@@ -6,7 +6,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - astropy>=5,<6
```

Comment:
This file contains the dependencies for development. There's usually no benefit to allow old versions here. They should be allowed in the `pyproject.toml` though. (However, we are currently not testing for such old versions, are we?)